src/styles/shared/components/buttons/_index.scss

```diff
@@ -1,16 +1,2 @@
-@use './mixins';
-
-// A button with text, and optionally an icon
-.LabeledButton {
-  @include mixins.LabeledButton;
-}
-
-// A button with only an icon and no label/text
-.IconButton {
-  @include mixins.IconButton;
-}
-
-// A button styled to appear as a link, with underline on hover
-.LinkButton {
-  @include mixins.LinkButton;
-}
+@forward 'config';
+@forward 'mixins';
```

Comment:
I tend to be a little wary about wildcard forwarding from modules (ie. `export * from <module>` in JS or `@forward <module>` in SASS) because it makes the public API of the package less explicit. How cumbersome would it be here to just explicitly export the mixins or config variables that are intended for external use?

Reply:
@robertknight 

There are two additional things we could do here!

* `@forward 'mixins' as c-*;` — prefix the result
* `@forward 'mixins' show mixin-1, mixin-2, ...;` or `@forward 'mixins' hide mixin-4;` — selective member visibility

The prefixing seems less cumbersome.

Reply:
As the goal here was to be explicit about the public API of the package the second solution is more aligned with what I had in mind.

Reply:
OK, surely

Reply:
@robertknight I've pushed a new commit that clearly defines the API: https://github.com/hypothesis/client/pull/3266/commits/69a777ae01f8d1b4b7bb7760ac8df6e1e7a6e78b

Reply:
Thanks - this looks good.